pkg/cloud/azurestack/azurestack.go

```diff
@@ -2,17 +2,21 @@ package azurestack
 
 import (
 	"embed"
+	"encoding/json"
 	"fmt"
 
 	"github.com/asaskevich/govalidator"
+	configv1 "github.com/openshift/api/config/v1"
 	appsv1 "k8s.io/api/apps/v1"
+	azure "sigs.k8s.io/cloud-provider-azure/pkg/provider"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/openshift/cluster-cloud-controller-manager-operator/pkg/cloud/common"
 	"github.com/openshift/cluster-cloud-controller-manager-operator/pkg/config"
 )
 
 const providerName = "azurestack"
+const vmTypeStandard = "standard"
```

Comment:
Not exported in the Azure API somewhere?

Reply:
i couldn't find one, sadly

Reply:
with fresh eyes, i have found it